core/src/main/java/discord4j/core/shard/GatewayBootstrap.java

```diff
@@ -267,8 +284,7 @@
      *
      * @param memberRequest {@code true} if enabling the large guild member requests, {@code false} otherwise
      * @return this builder
-     * @see
-     * <a href="https://discordapp.com/developers/docs/topics/gateway#request-guild-members>Request Guild Members</a>
+     * @see <a href="https://discordapp.com/developers/docs/topics/gateway#request-guild-members>Request Guild Members</a>
```

Comment:
This is missing a closing `"`. This typo already exists in `master` but you could fix it to recover syntax highlighting in this file.